docs/cn/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -1,128 +1,1031 @@
+
 ---
 layout: global
-title: 在Kubernetes上运行Alluxio
+title: 在Kubernetes上部署Alluxio
 nickname: Kubernetes上的Alluxio
 group: Install Alluxio
 priority: 4
 ---
 
-Alluxio可以在Kubernetes上运行。这个指南演示了如何使用Alluxio Github库中的说明来在Kubernetes上运行Alluxio。
+Alluxio可以在Kubernetes上运行。本指南演示了如何使用Docker映像或`helm`中包含的规范在Kubernetes上运行Alluxio
+
+*目录
+{:toc}
+
+##先决条件
 
-# 基础教程
+一个Kubernetes集群(版本> = 1.8)。在默认规范下，Alluxio workers可以通过设置`sizeLimit`参数来决定`emptyDir`卷的大小。这是Kubernetes 1.8版本中的一个Alpha特性。在使用前请确保此功能已启用。
 
-本教程将介绍Kubernetes上的基本Alluxio安装。
+一个Alluxio Docker镜像[alluxio/{{site.ALLUXIO_DOCKER_IMAGE}}](https://hub.docker.com/r/alluxio/{{site.ALLUXIO_DOCKER_IMAGE}}/)。如果使用私有Docker注册表，请参阅Kubernetes [documentation] (https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/)。
+确保[Kubernetes网络策略](https://kubernetes.io/docs/concepts/services-networking/network-policies/)允许应用程序(Alluxio客户端)和Alluxio Pods之间在已定义端口上的连接。
 
-## 前提条件
+##基本设置
 
-- 一个Kubernetes集群 (版本 >= 1.8). Alluxio workers将使用`sizeLimit`参数来限制使用`emptyDir`卷的大小。这是Kubernetes 1.8的一个alpha特性。
-请确保该功能已启用。
-- 一个Alluxio Docker镜像。请参阅[用Docker运行Alluxio]({{ '/cn/deploy/Running-Alluxio-On-Docker.html' | relativize_url }})了解如何构建镜像。该镜像必须可以从运行Alluxio进程的所有Kubernetes主机提取。这可以通过将镜像推送到可访问的Docker注册表来实现，或者将镜像单独推送到所有主机。如果使用私人Docker注册表，请参阅Kubernetes [文档](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/)。
+本教程介绍了在Kubernetes上的基本Alluxio安装。 Alluxio支持在Kubernetes上两种安装方法:使用[helm](https://helm.sh/docs/)图表或使用`kubectl`。如果可选，`helm`是首选安装Alluxio方法。如果没法使用`helm`安装或需要额外定制化部署，则可以直接通过原生Kubernetes资源规范使用`kubectl`。
 
-## 克隆Alluxio库
+>注意：从Alluxio 2.3起，Alluxio仅支持helm 3。
+>参阅如何[从helm 2迁移到helm 3](https://helm.sh/docs/topics/v2_v3_migration/)。
+
+{% accordion setup %}
+  {% collapsible (Optional) Extract Kubernetes Specifications %}
+
+如果使用私有`helm` 仓库或使用原生Kubernetes规范，从Docker镜像中提取部署Alluxio所需的Kubernetes规范。
 
 ```console
-$ git clone https://github.com/Alluxio/alluxio.git
-$ cd integration/kubernetes
+$ id=$(docker create alluxio/{{site.ALLUXIO_DOCKER_IMAGE}}:{{site.ALLUXIO_VERSION_STRING}})
+$ docker cp $id:/opt/alluxio/integration/kubernetes/ - > kubernetes.tar
+$ docker rm -v $id 1>/dev/null
+$ tar -xvf kubernetes.tar
+$ cd kubernetes
 ```
 
-可以在`integration/kubernetes`下找到部署Alluxio所需的Kubernetes说明。
+ {% endcollapsible %}
+  {% collapsible (Optional) Provision a Persistent Volume %}
+注意：[嵌入式日志]({ {'/en/operation/Journal.html' | relativize_url}}＃embedded-journal-configuration)
```

Comment:
@thanosaw the jenkins build failed saying the following link is invalid
```
  /tmp/docsCheck588712490/docs/cn/deploy/Running-Alluxio-On-Kubernetes.md:
    138: relative link did not match expected pattern "[UFS Journal]({{'/en/operation/Journal.html' | relativize_url}}#ufs-journal-configuration)"
    160: relative link did not match expected pattern "[UFS Journal]({{'/en/operation/Journal.html' | relativize_url}}#ufs-journal-configuration)"
    459: relative link did not match expected pattern "[journal modes]({{'/en/operation/Journal.html' | relativize_url}})"
    638: relative link did not match expected pattern "[formatting journal with kubectl]({{'/en/deploy/Running-Alluxio-On-Kubernetes.html#format-journal-1' | relativize_url}})"
    641: relative link did not match expected pattern "[分层存储]({{'/en/core-services/Caching.html#multiple-tier-storage' | relativize_url}})"
    665: relative link did not match expected pattern "[Verify Alluxio]({{'/en/deploy/Running-Alluxio-Locally.html?q=verify#verify-alluxio-is-running' | relativize_url}})"
    706: relative link did not match expected pattern "[POSIX API]({{'/en/api/POSIX-API.html' | relativize_url}})"
    730: relative link did not match expected pattern "[此处]({{'/en/deploy/Running-Alluxio-On-Kubernetes.html#deploy-using-helm' | relativize_url}})"
    939: relative link did not match expected pattern "[metrics json]({{'/en/operation/Metrics-System.html' | relativize_url}})"
    940: relative link did not match expected pattern "[fsadmin metrics CLI]({{'/en/operation/Admin-CLI.html' | relativize_url}})"
```
The link needs to follow the strict rules.
For example
```
valid -> [UFS Journal]({{ '/en/operation/Journal.html' | relativize_url }}#ufs-journal-configuration)
not valid -> [UFS Journal]({{'/en/operation/Journal.html' | relativize_url}}#ufs-journal-configuration)
```
space is removed after `({{`

Reply:
Usually, in Chinese doc, we prefer to link to the other Chinese docs instead of English docs. 
For example, `"[metrics json]({{'/en/operation/Metrics-System.html' | relativize_url}})"`
has corresponding docs under `docs/cn/operation/Metrics-System.md`, then we can change the `en` in the link to `cn`. This can be done in future PRs since not all the links have corresponding chinese docs already

Reply:
Sorry about that, still learning the syntax of Markdown. Won't happen again 👍